pypdf/generic/_viewerpref.py

```diff
@@ -69,6 +75,10 @@ def _get_int(self, key: str, deft: Optional[NumberObject]) -> NumberObject:
     def _set_int(self, key: str, v: int) -> None:
         self[NameObject(key)] = NumberObject(v)
 
+    @property
+    def PRINTINGSCALING(self) -> NameObject:
```

Comment:
Why do we have a completely uppercase property for this? Shouldn't we use `print_scaling` as for the actual object name instead?

Reply:
this is a constant like property. this is why it is in uppercase